pyro/distributions/torch.py

```diff
@@ -1,15 +1,87 @@
 from __future__ import absolute_import, division, print_function
 
+import math
+
 import torch
 from torch.distributions import constraints, kl_divergence, register_kl
+from torch.distributions.multivariate_normal import _batch_mv
+from torch.distributions.lowrank_multivariate_normal import (_batch_lowrank_mahalanobis,
+                                                             _batch_lowrank_logdet)
+from torch.distributions.utils import _standard_normal
```

Comment:
Should we also override the kl implementations?

Reply:
Because we just introduce KL for SVI recently, it is unlikely to have some tests with kl affected by this issue. If the fix in pytorch is not able to be in 1.0, I agree with you that we should override KL too.